lib/ui/SpinniesManager.ts

```diff
@@ -282,4 +323,4 @@ class SpinniesManager {
   }
 }
 
-module.exports = new SpinniesManager();
+export default new SpinniesManager();
```

Comment:
Currently our tests are failing because `SpinniesManager.init()` is not a function. Am I exporting incorrectly here?

Reply:
So default exports are going to be tricky to convert because they are actually just a named export with the reserved keyword `default`.  So changing this to `export default new SpinniesManager` makes all of the CJS `require` statements would need to be updated to `const { default: SpinniesManager} = require('path/to/spinniesmanager');`

A workaround is to do something like:

```javascript
const toExport = new SpinniesManager();
export default toExport;
module.exports = toExport;
```

Then once the migration is fully complete we can walk this back and switch to ESM default exports

Reply:
Fixed in https://github.com/HubSpot/hubspot-cli/pull/1233/commits/df9fd03807a2561fac999dea1710b89bacb59030